gensrc/thrift/FrontendService.thrift

```diff
@@ -492,6 +492,7 @@ struct TStreamLoadPutRequest {
     14: optional string columnSeparator
 
     15: optional string partitions
+    16: optional string enable_hub
```

Comment:
You should use `boolean` instead of `string` to a switch filed

However, I think you'd better not to add this filed.